content/source/docs/enterprise/run/index.html.md

```diff
@@ -42,7 +42,9 @@ Speculative plans are plan-only runs: they show a set of possible changes (and c
 
 There are three ways to run speculative plans:
 
-- In [VCS-backed workspaces](./ui.html), each pull request starts a speculative plan. TFE adds a link to the plan in the VCS provider's pull request interface. If multiple workspaces use the same repository, each of them will add a plan to the pull request.
+- In [VCS-backed workspaces](./ui.html), a pull request to the linked VCS branch (from another branch in the same repository) starts a speculative plan. Each workspace linked to the destination branch will perform a plan and post a link to it in the VCS provider's pull request interface.
```

Comment:
I wonder if this would read more clearly without the parens?

Reply:
I went back and forth. :/